src/test/fixtures/blueprints.ts

```diff
@@ -35,3 +37,49 @@ export const emptyGetBlueprints: GetBlueprintsApiResponse = {
   meta: { count: 0 },
   data: [],
 };
+
+export const mockBlueprintComposes: GetBlueprintComposesApiResponse = {
```

Comment:
Let's turn this into a function that returns a different set of images depending on the id of the blueprint.

For 'Dark Chocolate' we can return these two.

For 'Milk Chocolate' we can return a different response, with 0 images. That way we can easily test the state where a blueprint is selected but that blueprint has no images associated with it.

Reply:
Makes sense.

I suppose we will just render an empty table for "Milk Chocolate" for now. We could just verify that the table elements have disappeared in the test, for now.